nominal/experimental/video_processing/video_conversion.py

```diff
@@ -46,6 +51,7 @@ def normalize_video(
                   can impact performance negatively-- typically, a value at or around 2s is considered
                   "best of both worlds" as a reasonable default value.
         force: If true, forcibly delete existing output path if already exists.
+        resolution: If provided, re-scale the video to the provided resolution
```

Comment:
For my own understanding: is the idea that users will discover valid specifiers via their editors interpreting the type annotations? Or, to put it differently, what is the policy on documenting the different ways you can specify an argument, vs expecting users to be able to discover it.

In this case, e.g., there are roughly three modes:

- `"720p"`, `"1080p"`, etc.
- `VideoResolution(resolution_height=320, resolution_width=240)`
- Or upscaling: `VideoResolution(resolution_height=1024, allow_upscaling=True)`